src/ziggurat/kafka_consumer/consumer_handler.clj

```diff
@@ -77,16 +81,17 @@
 
 (defn poll-for-messages
   [^Consumer consumer handler-fn topic-entity consumer-config]
-  (try
-    (loop [records []]
-      (when (not-empty records)
-        (let [batch-payload (create-batch-payload records topic-entity)]
-          (process handler-fn batch-payload)))
-      (recur (seq (.poll consumer (Duration/ofMillis (or (:poll-timeout-ms-config consumer-config) DEFAULT_POLL_TIMEOUT_MS_CONFIG))))))
-    (catch WakeupException e
-      (log/errorf e "WakeupException while polling for messages for: %s" topic-entity))
-    (catch Exception e
-      (log/errorf e "Exception while polling for messages for: %s" topic-entity))
-    (finally (do (log/info "Closing the Kafka Consumer for: " topic-entity)
-                 (.close consumer)))))
+  (clog/with-logging-context {:consumer-group topic-entity}
```

Comment:
Why does this log look empty? As in why is there no log message of some sort?

Reply:
No no. This is just us setting a context so that any logs if printed inside the context, will have consumer-group as a key, by default.